### DISCUSSION TITLE
AniMatrix some led always on

### DISCUSSION THREAD

**Discussion by JV-bug** [seerge/g-helper#2681] [Q&A]
If i disable animatrix, it goes off for some couple of minutes, then couple of leds do stay always on, no matter what, some pics attached.
![image](https://github.com/seerge/g-helper/assets/67544649/3b396e4a-5018-42b7-b3a0-a1c19724067f)

![image](https://github.com/seerge/g-helper/assets/67544649/842738ce-497e-4970-9c8f-b23fc0287dba)

**Comment by seerge** [comment]
@JV-bug hello, it’s not software related.

It’s just 2 stuck LEDs on hardware

**Reply by JV-bug** [reply]
but why when i initially press animatrix off in ghelper, they are all off and then after some minutes, those leds turn on themselves, tested on armoury crate, are always off

**Reply by seerge** [reply]
G-Helper is not setting those LEDs on. When "Off" is selected in a dropdown it does nothing at all.

Does turning matrix on / off change leds somehow ? Or a custom picture with just a black square? :)

**Reply by JV-bug** [reply]
when i enable animatrix those led function ok, going on and off depending on the "picture" or "mode" used, then i set animatrix to off, and the lid is totally dar, no leds are on, then after 2-5 minutes they go on again, while animatrix is set to off.
![media_20240604_134036_8334423714968455427](https://github.com/seerge/g-helper/assets/67544649/a791d349-cf0f-47dc-9b94-bf0e1bc009c3)
![media_20240604_134036_1458362584626122275](https://github.com/seerge/g-helper/assets/67544649/27a93244-7ea1-4ee2-8efc-004535f37bfe)

**Reply by seerge** [reply]
@JV-bug try to quit app after you have turned them off. Do they appear? :)

**Reply by JV-bug** [reply]
no leds are on now, despite g-helper is still running.

**Reply by seerge** [reply]
@JV-bug so it’s all good now?

P.S. Do you have any AC leftovers running in background by any chance ? In particular Asus / Aura Lighting service (it can be called a bit differently on different systems) .

**Reply by JV-bug** [reply]
all service show as stopped via extra's menu in g-helper, but still for now all leds are off.

**Reply by seerge** [reply]
@JV-bug extra menu in g-helper does not stop armoury leftovers, that's why I have asked.

But I'm glad it's off :)
